components/ai_chat/resources/untrusted_conversation_frame/components/conversation_entries/index.tsx

```diff
@@ -159,14 +171,19 @@ function ConversationEntries() {
                       }
                     />
                   )}
-                  {isAIAssistant && !showEditInput && (
-                    <AssistantResponse
-                      entry={latestTurn}
-                      isEntryInProgress={isEntryInProgress}
-                      allowedLinks={allowedLinksForTurn}
-                      isLeoModel={conversationContext.isLeoModel}
-                    />
-                  )}
+                  {isAIAssistant && !showEditInput &&
+                    group.map((entry, i) => {
+                      const isThisGroupInProgress = isEntryInProgress && (i === group.length - 1)
+                      const isThisGroupActive = isLastEntry && (i === group.length - 1)
```

Comment:
nit: I think this is talking about the entry, not the group
```suggestion
                      const isThisEntryInProgress = isEntryInProgress && (i === group.length - 1)
                      const isThisEntryActive = isLastEntry && (i === group.length - 1)
```